src/components/utils/JobCard.tsx

```diff
@@ -150,29 +150,35 @@ export default class JobCard extends React.Component<IProps, IState> {
                     ) : (
                         ""
                     )}
-                    <div className="d-flex mt-2" style={{ height: "1.5rem" }}>
-                        <ProgressBar
-                            className="text-darker font-weight-bold flex-grow-1 h-unset"
-                            animated={!job.stoppedAt}
-                            label={
-                                typeof job.progress === "number"
-                                    ? `${job.progress.toString()}%`
-                                    : undefined
-                            }
-                            now={typeof job.progress === "number" ? job.progress : 100}
-                            striped
-                            variant={variant}
-                        />
-                        {job.canCancel && !job.stoppedAt ? (
-                            <Button
-                                style={{ padding: "0 1em" }}
-                                className="ml-1"
-                                variant="danger"
-                                onClick={() => this.props.onCancel(job)}>
-                                <FontAwesomeIcon icon="times" className="d-block" />
-                            </Button>
-                        ) : null}
-                    </div>
+
+                    {/* PROGRESS BAR */}
+                    {job.stoppedAt == undefined ? (
+                        <div className="d-flex mt-2" style={{ height: "1.5rem" }}>
+                            <ProgressBar
+                                className="text-darker font-weight-bold flex-grow-1 h-unset"
+                                animated={!job.stoppedAt}
+                                label={
+                                    typeof job.progress === "number"
+                                        ? `${job.progress.toString()}%`
+                                        : undefined
+                                }
+                                now={typeof job.progress === "number" ? job.progress : 100}
+                                striped
+                                variant={variant}
+                            />
+                            {job.canCancel && !job.stoppedAt ? (
+                                <Button
+                                    style={{ padding: "0 1em" }}
+                                    className="ml-1"
+                                    variant="danger"
+                                    onClick={() => this.props.onCancel(job)}>
+                                    <FontAwesomeIcon icon="times" className="d-block" />
+                                </Button>
+                            ) : null}
+                        </div>
+                    ) : (
+                        ""
+                    )}
```

Comment:
Just do `null` instead of `("")`

Reply:
copying the behavior for the others, otherwise I'd use undefined